### PR TITLE
change code and documentation for read_habitatmap_stdized and read_wa…

### PR DESCRIPTION
**Pull request by ToonHub**
…tersurfaces_hab as we no longer use the term 'patch'

The adapted functions will no longer work with older versions of habitatmap_stdized and watersurfaces_hab. It will give an error.

Should we change the function so it can also read older versions (that contain the layer 'habitatmap_stdized_patches') and give a warning if older versions are read?

### CODE REVIEW

File: R/read_habitatdata.R [hunk 1]
```diff
@@ -458,18 +449,18 @@ read_habitatmap <-
 #' \href{https://github.com/inbo/n2khab-preprocessing}{n2khab-preprocessing}
 #' repository.
 #'
-#' @param keep_aq_patches Logical; \code{TRUE} by default.
+#' @param keep_aq_types Logical; \code{TRUE} by default.
 #' The data source \code{habitatmap_terr} aims at delineating all
 #' polygons with at least one (semi-)terrestrial type (i.e. patch).
```
Comment (florisvdh) [inline R/read_habitatdata.R]:
```suggestion
#' polygons with at least one (semi-)terrestrial type.
```

**Comment by florisvdh** [comment]
BTW, in `DESCRIPTION` you best update the dev version number to `0.0.3.9037` (37 being the PR number)

File: R/read_habitatdata.R [hunk 2]
```diff
@@ -449,7 +440,7 @@ read_habitatmap <-
 #' \href{https://doi.org/10.5281/zenodo.3468948}{Zenodo}, that contains:
 #' \itemize{
 #'   \item{\code{habitatmap_terr_polygons}: a spatial polygon layer}
-#'   \item{\code{habitatmap_terr_patches}: a table in which every row
+#'   \item{\code{habitatmap_terr_types}: a table in which every row
 #'   corresponds with one patch.}
```
[the remark this answers is not in the document]
Comment (ToonHub) [reply R/read_habitatdata.R]: I did not intend to change the function read_habitat_terr yet. Better first create the new version of habitatmap_terr

File: R/read_habitatdata.R [hunk 3]
```diff
@@ -487,7 +478,7 @@ read_habitatmap <-
 #'     \item \code{source}: states where \code{description} comes from: either
 #'     \code{habitatmap_stdized} or \code{habitatmap_stdized + interpretation}
 #'   }
-#'   \item \code{habitatmap_terr_patches}: a tibble with the following
+#'   \item \code{habitatmap_terr_types}: a tibble with the following
 #'   variables (the first 5 being identical to those in
```
[the remark this answers is not in the document]
Comment (florisvdh) [reply R/read_habitatdata.R]: @ToonHub can you still address the above comment? (delete line with `patch_id`)
Further, you'll need to auto-update the Rd files as well.

**Comment by ToonHub** [comment]
I did 1. en 2. @florisvdh can you check the new versions on Zenodo.
After we do 3., in Zenodo we will have tot refer to the commit corresponding with the new versions

**Comment by florisvdh** [comment]
I'm looking at it. Can you also make a pull request in n2khab-preprocessing, regarding the removal of 7220?

**Comment by florisvdh** [comment]
@ToonHub I checked both gpkg files from Zenodo, they are OK! I emailed you some remarks on metadata.

One addition is proposed in PR #41

I'm in the process of uploading the new version of `habitatmap_terr`, I will keep you informed.

**Comment by florisvdh** [comment]
@ToonHub you can have a look if the metadata of https://doi.org/10.5281/zenodo.3540740 are OK to you (note, @w-jan already reviewed the actual data). After that, this PR can be merged to master!

**Comment by ToonHub** [comment]
OK!

**Pull request merged**